docs/source/usage/examples.rst

```diff
@@ -7,22 +7,23 @@ Create a dictionary to handle communication with the plug-in:
 .. code-block:: python
 
     from zowe.zos_console_for_zowe_sdk import Console
-    connection = {
-        "host_url": "'<host address>'",
+    profile = {
+        "host": "<host address>",
+        "port" : 443 , # Include the port if different from the default (443)
         "user": "<user>",
         "password": "<password>",
     }
 
-    my_console = Console(connection)
+    my_console = Console(profile)
 
 Alternatively you can use an existing Zowe CLI profile instead:
 
 .. code-block:: python
 
-    from zowe.zos_console_for_zowe_sdk import Console
+  from zowe.zos_console_for_zowe_sdk import Console
+  from zowe.zos_core_for_zowe_sdk import ProfileManager
```

Comment:
Sorry one more thing that I missed - the name of this package is `zowe.core_for_zowe_sdk` (without the "zos")